baguetteBox.css

```diff
@@ -1,199 +1,199 @@
-@CHARSET "UTF-8";

```

Comment:
You had a space at the beginning of the filename on baguetteBox.css, which was causing it not to load.  I have renamed it here.